src/styles/blocks/compare-bikes.scss

```diff
@@ -0,0 +1,46 @@
+.compare-bike {
+  font-family: "Poppins", sans-serif;
+  color: $first-text-colour;
+
+  &__item:not(:last-child) {
+    margin-bottom: 40px;
+
+    @include tablet {
+      margin-bottom: 48px;
+    }
+
+    @include desktop {
+      margin-bottom: 0;
+    }
+  }
+
+  &__photo {
+    width: 100%;
+    margin-bottom: 32px;
+    transition: transform 0.3s;
+
+    &:hover {
+      transform: scale(1.1);
+    }
+  }
+
+  &__title {
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 140%;
```

Comment:
these properties you use in several css rules. Create `placeholder` instead or create one common class for similar elements and use `mixin`